.github/workflows/test.yml

```
@@ -0,0 +1,47 @@
+        echo "Code coverage: $coverage%"
+        # Check coverage threshold
+        if [ $coverage -lt $MIN_COVERAGE ]; then
+          echo "::error::Code coverage below ${MIN_COVERAGE}% (current: ${coverage}%)"
+          exit 1
+        fi
+
+        make test
+        sui move coverage summary
+        
+    - name: Check coverage
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)